### PR TITLE
[test] Port JacobiNoToolHybrid check to Alps

### PR DESCRIPTION
**Pull request by jgphpc**

### CODE REVIEW

**Review by kraushm — APPROVED** [review]
lgtm

**Review by teojgo — CHANGES_REQUESTED** [review]
Can you change the `"` to `'` to be consistent with our other tests?

**Comment by jgphpc** [comment]
@teojgo I would rather not: https://black.readthedocs.io/en/stable/the_black_code_style.html#strings

**Comment by teojgo** [comment]
> @teojgo I would rather not: https://black.readthedocs.io/en/stable/the_black_code_style.html#strings

In all the other parts of the code we use single quotes.

**Comment by vkarak** [comment]
@jgphpc We do have a coding style in ReFrame which imposes single quotes.

**Comment by vkarak** [comment]
https://github.com/eth-cscs/reframe/wiki/Coding-Style-Guide#formatting-strings

**Comment by jgphpc** [comment]
@vkarak then why the ci does not fail ?

**Comment by vkarak** [comment]
It can't catch everything and as far as I remember there is not option to impose one or the other style in the PEP8 checks. But even if there were one, it wouldn't be practical, because there are enough cases that the double quotes must be used in order to avoid ugly escapes. So this is one of the rules that we have to respect as we type. Similarly, the blank line after a code block cannot be imposed by the style checker we are using, but we have to respect it.

**Review by teojgo — CHANGES_REQUESTED** [review]

File: cscs-checks/tools/profiling_and_debugging/notool.py [hunk 1]
```diff
@@ -7,25 +7,30 @@
...
 @rfm.parameterized_test(['C++'], ['F90'])
 class JacobiNoToolHybrid(rfm.RegressionTest):
     def __init__(self, lang):
         super().__init__()
         self.descr = f'Jacobi (without tool) {lang} check'
-        self.name = f'{type(self).__name__}_{lang.replace("+", "p")}'
-        self.valid_systems = ['daint:gpu', 'daint:mc', 'dom:gpu', 'dom:mc']
-        self.valid_prog_environs = ['PrgEnv-cray', 'PrgEnv-cray_classic',
-                                    'PrgEnv-gnu', 'PrgEnv-intel', 'PrgEnv-pgi']
+        self.name = f'{type(self).__name__}_{lang.replace('+', 'p')}'
```
[the remark this answers is not in the document]
Comment (jgphpc) [reply cscs-checks/tools/profiling_and_debugging/notool.py]: https://github.com/eth-cscs/reframe/pull/1713/commits/3bdf2eda3c82cbdd1871ff6ab7ffb109a80de763

**Review by teojgo — APPROVED** [review]

**Review by vkarak — APPROVED** [review]
lgtm

**Pull request merged**